addons/product/models/res_partner.py

```diff
@@ -11,7 +11,7 @@ class Partner(models.Model):
     # NOT A REAL PROPERTY !!!!
     property_product_pricelist = fields.Many2one(
         'product.pricelist', 'Pricelist', compute='_compute_product_pricelist',
-        inverse="_inverse_product_pricelist", company_dependent=False,
+        inverse="_inverse_product_pricelist",
```

Comment:
can be left as a warning that something ~shitty~ fishy is going on

Reply:
The depends_context('company') is already there to say that, but ok ^^.